enhancements/installer/telemeter-metrics.md

```diff
@@ -0,0 +1,430 @@
+# TYPE cluster_installation_gather counter
+cluster_installation_gather{platform="aws",result="success",version="4.2",os="linux"} 1
+```
+Potential label values would be the same as `cluster_installation_manifests` except `result` could return an extra value for `SSHError` which shows that users tried to connect to gather, but were unauthorized.
+
+#### Duration
+
+The `cluster_installation_create` metric above captures the overall duration of an installation. The metrics in this category represent the duration of  each stage of installation in the execution of the `create` or `wait-for` commands. The proposed metrics are: 
+
+* `cluster_installation_duration_infra`
+* `cluster_installation_duration_api`
+* `cluster_installation_duration_bootstrap`
+* `cluster_installation_duration_operators`
+
+Each of these metrics (except infrastructure) would have five labels:
+  * `command` - either `create` or `wait-for`
+  * `result` - `success` or `error`
+  * `version` 
+  * `platform`
+  * `le` - step size of 3 minutes with a max of 30
+
+An example of a sample tracking how long it took for bootstrap complete after the temporary control plane has come up: 
+```
+# HELP cluster_installation_bootstrap tracks the duration between the appearance of the temporary control plane and bootstrap completion.
+# TYPE cluster_installation_bootstrap histogram
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="3"} 0
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="6"} 0
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="9"} 0
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="12"} 1
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="15"} 1
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="18"} 1
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="21"} 1
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="24"} 1
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="27"} 1
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="30"} 1
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="+Inf"} 1
+cluster_installation_bootstrap_sum{command="create",platform="aws",result="success"} 10
+cluster_installation_bootstrap_count{command="create",platform="aws",result="success"} 1
+```
+
+#### Modification
+The `cluster_installation_modification`- metrics represent whether user-modified manifest or ignition-config assets were used to install the cluster.
+
+The proposed metrics represent categories of one or more asset file modified during an installation:
+
+Metric | Modified Files
+-------|---------------
```

Comment:
@abhinavdahiya  could you give some feedback on what you think about these categories?

Reply:
why not to have this files as a tag instead? It would be much easier than maintain those categories over time and the number of files anyhow limited - so cardinatlity wise u either get different metric name or different tag. 
U can just submit metric per file as tag and that's it. Much easier to maintain

Reply:
I don't understand what a "tag" is in this context. Can you elaborate?

Reply:
sorry, label